docs/source/regenerate_apiref.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
```

Comment:
Can we create `utils/{apiref.py, notebook.py}` and replace support code into this files